solr/core/src/java/org/apache/solr/api/AnnotatedApi.java

```diff
@@ -62,6 +63,8 @@
 
 public class AnnotatedApi extends Api implements PermissionNameProvider , Closeable {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  private static final ObjectMapper mapper = SolrJacksonAnnotationInspector.createObjectMapper()
```

Comment:
Oh, I didn't notice before that it was created for each request. That was horrible!

Still, we already have one instance here and another (configured the same) in `ContainerPluginsRegistry`. We should probably expose this instance here and reuse it in `ContainerPluginsRegistry`.

Reply:
It was not created per request. it was created per API